### ISSUE TITLE
Question

### ISSUE THREAD

**Issue by Truyn** [Stewart-Anubis/Sm-Zombiereloaded-3-Franug-Anubis-Edition#5]
Why don't you add this version's improvements to the franug version?

I would love to put your version, but you do not support it

**Comment by Truyn** [comment]
Just wanted to know you as an understanding person.

Why does the official version still require zr_classfix

**Comment by Truyn** [comment]
it's not serious, why doesn't anyone implement other save methods?
Why not add support for Mysql or at least fix a bug that requires zr_classfix

**Comment by Truyn** [comment]
There is also still a bug that the zombie does not appear in the first round.
Neither mp_restartgame nor mp_warmup helps

**Comment by Truyn** [comment]
I like your work, I think you could do more

**Comment by Stewart-Anubis** [comment]
Good night, I stopped updating the "Sm-Zombiereloaded-3-Franug-Anubis-Edition" because I realized that the more functions or plugins integrated, the greater the size of the plugin and its complexity, making it difficult to edit and fix any bugs.
If you look closely, heavy plugins like “shop” are generally made lighter using just one “Core” and the functions are handled by separate “modular” plugins.
When plugins become too large, loading failures and bugs related to server delays can occur, such as the “zr_classfix” requirement. At the end of the round there are so many functions executing the hook at the same time that it causes the Zombiereloaded class error bug. Added a 1 second timer for Zombiereloaded to move players to CT after Round_end, this fixed at least for me the need for “zr_classfix”. In my tests the bug disappeared, but it has to be tested with many players online at the same time, which I'm not able to do at the moment. This modification already added the Franug version. The aim is to make Zombiereloaded modular and as light as possible, to fit the most diverse server needs.
The error of not appearing the first zombie can be several things such as zr_forceteams.
My test server is online 24h, but it is empty, if there were more players online, I could fix more bugs that only present with more than 20 or 30 players. IP 177.37.113.17:27015
